.circleci/config.yml

```diff
@@ -43,6 +53,15 @@ jobs:
     steps:
       - test
 
+  final-catch-all:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - run:
+          name: Success
+          command: |
+            exit 0
```

Comment:
identical to `command: exit 0` I think?

Reply:
yeah, was just copy-pasta the example code